backend/apps/volontulo/views/api.py

```diff
@@ -65,6 +69,62 @@ def login_view(request):
     )
 
 
+@api_view(['POST'])
+@authentication_classes((CsrfExemptSessionAuthentication,))
+@permission_classes((AllowAny,))
+def register_view(request):
+    """REST API register view."""
+    if not request.user.is_authenticated():
+        email = request.data.get('email')
+        password = request.data.get('password')
+
+        try:
+            user = User.objects.create_user(
+                username=email,
+                email=email,
+                password=password,
+                is_active=False
+            )
+
+            user.save()
+            profile = UserProfile(user=user)
+            ctx = {'token': profile.uuid, 'angular_root':
```

Comment:
AFAIR @juliaanholcer already add `ANGULAR_ROOT` to all emails context, so there's no need to add it explicitely here.

Also it will  be great to format it a little more sanely.